docker-app/qfieldcloud/subscription/models.py

```diff
@@ -889,6 +889,37 @@ def create_subscription(
 
         return trial_subscription_obj, regular_subscription_obj
 
+    def clean(self):
+        """
+        Validates that the subscription's active period does not overlap with
+        any other active subscriptions for the same account.
+        """
+        # If there is no `active_since`, nothing to check.
+        if not self.active_since:
+            return
+
+        conflicts = (
+            self.__class__.objects.filter(
+                account=self.account,
+            )
+            .exclude(pk=self.pk)
+            .filter(
+                Q(active_since__lt=self.active_since)
+                & (Q(active_until__isnull=True) | Q(active_until__gt=self.active_since))
+            )
+        )
+
+        if conflicts.exists():
+            raise ValidationError(
+                {
+                    "active_since": "This account already has an active subscription that overlaps with this period. Please cancel the existing subscription or choose a different date range."
```

Comment:
```suggestion
                    "active_since": _("This account already has an active subscription that overlaps with this period. Please cancel the existing subscription or choose a different time range.")
```

We need to translate. Also we talk about time range, not only date.

Reply:
Yes, indeed, thanks!